src/app/shell/main/main.component.ts

```diff
@@ -29,12 +29,17 @@ export class MainComponent implements OnInit {
 
   constructor(private store: Store) { }
 
+  
   ngOnInit(): void {
     this.store.dispatch([new GetCategories(),
     new GetCategoriesIcons(),
     new GetPopWorkshops(),
     new ChangePage(true)
     ]);
+    
+    this.cards$.subscribe(cards =>
+      this.cards = cards
```

Comment:
what the reason for this change?

Reply:
I thought that the structure of all these components that contains workshop should  have the same method of realizing cards.

Reply:
use cards$ directly from the store through async pipe